### ISSUE TITLE
Plan and next release date

### ISSUE THREAD

**Issue by harveyhgodfrey** [RestyaPlatform/board#1472]
We are very much interested in this product. BUT..

We are very concerned about releases and activities. Last release is November. No proper milestones and activities 

Interested to know commitments and milestones of thsi project

**Comment by ssivachidambaram** [comment]
@harveyhgodfrey 

Thanks for your interest. We are working on our next release process and very soon we will release the next version. Also we will update our next commitments and milestones after the current release.

**Comment by harveyhgodfrey** [comment]
unportunately no change 

no improvment no proper rleases and milestones

**Comment by ssivachidambaram** [comment]
@harveyhgodfrey 

Yes. Sorry for that. We are also internally planning to streamline the milestone process. Once milestone process streamlined, we will update here. So only we have keep this issue as open.
